sdk/scheduler/src/test/java/com/mesosphere/sdk/scheduler/DefaultSchedulerTest.java

```diff
@@ -872,6 +872,16 @@ private void statusUpdate(Protos.TaskID launchedTaskId, Protos.TaskState state)
         taskIds.add(installStep(1, 0, getSufficientOfferForTaskB()));
         taskIds.add(installStep(1, 1, getSufficientOfferForTaskB()));
 
+        while (true) {
```

Comment:
Yeah, the loop was more complicated at some point before I checked it in. Good catch.